<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>791</width>
      <height>571</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>4</number>
    </property>
    <widget class="QWidget" name="StartPage">
     <widget class="QPushButton" name="startButton">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>290</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Начать тест</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Question1">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>90</y>
        <width>231</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Какой ваша любимая расцветка</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton1">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>140</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Синежелтый</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton2">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>180</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Красночерный</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton3">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>230</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Желточерный</string>
      </property>
     </widget>
     <widget class="QPushButton" name="nextButton1">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>280</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Далее</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Question2">
     <widget class="QPushButton" name="nextButton2">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>290</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Далее</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton6">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>230</y>
        <width>95</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>мы</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton5">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>190</y>
        <width>95</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>ты</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton4">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>150</y>
        <width>95</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>я</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>100</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Буквально...?</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Question3">
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>304</x>
        <y>120</y>
        <width>261</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Лучший отряд во время 2 мировой войны</string>
      </property>
     </widget>
     <widget class="QPushButton" name="nextButton3">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>340</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Далее</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton7">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>180</y>
        <width>95</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>отряд 731</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton8">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>230</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>отряд Солнышко</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton9">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>280</y>
        <width>95</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>отряд 14/88</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Results">
     <widget class="QLabel" name="resultLabel">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>120</y>
        <width>271</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Ваш результат</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
